package.json

```
@@ -80,6 +82,7 @@
   "copyright.owner": "IBM Corp.",
   "husky": {
     "hooks": {
+      "pre-commit": "if-env LINT_STAGED=false && echo \"lint-staged disabled via LINT_STAGED env var\" || lint-staged",
```

Let's use `0` instead of `false`, it's shorter to type :)

```suggestion
      "pre-commit": "if-env LINT_STAGED=0 && echo \"lint-staged disabled via LINT_STAGED env var\" || lint-staged",
```

---

Updated